test/colony-funding.js

```diff
@@ -682,91 +704,202 @@ contract("Colony Funding", addresses => {
       await colony.startNextRewardPayout(otherToken.address);
     });
 
+    it("should not be able to finalize the payout if payout is not active", async () => {
```

Comment:
This title is a little confusing as the finalization happens when the payout is not active, I think it should read `should not be able to finalize reward payout twice`